rpcperms/interceptor.go

```diff
@@ -94,6 +105,13 @@ var (
 		"/lnrpc.State/SubscribeState": {},
 		"/lnrpc.State/GetState":       {},
 	}
+
+	// allowRemoteSignerWhitelist defines methods that we allow to be called
+	// when the when we are waiting for the remote signer to connect, i.e.
```

Comment:
Nit for myself for next iteration:
```suggestion
	// when we are waiting for the remote signer to connect, i.e.
```